cypress/integration/project_registration.spec.js

```diff
@@ -203,6 +213,30 @@ context("project registration", () => {
           `project-${project1.name}`,
           "registered"
         ).should("exist");
+
+        // Make sure the transaction that the proxy returns contains the same
+        // cost details as the one we submitted.
+
+        // Force the transaction center to fetch all transactions.
+        cy.reload();
```

Comment:
This is necessary because the frontend polls for transactions at longer intervals and Cypress times out normally after 5s. By reloading the page we trigger the front-end to poll all transactions at once and can check for the relevant transaction immediately.

Reply:
I see why this is an issue, I ran into the same problem in another branch. See a17d58c - if the ui waits for the return of that call the center should be up-to-date and no reloading should be required.

Reply:
🍒 picked it: https://github.com/radicle-dev/radicle-upstream/pull/457/commits/055e33afd1ee258408d9cdaa35e9572d9c74c0d7